org-code-javabuilder/theater/src/test/java/org/code/theater/SceneTest.java

```diff
@@ -65,7 +64,7 @@ public void testPause() {
 
   @Test
   public void testDrawImage() {
-    final Image image = mock(Image.class);
+    final Image image = new Image(1, 1);
```

Comment:
This was easier than other ways I could think of to confirm that the image being added to the action list is a copy of the existing image (assert that `new Image(Image source)` constructor was called? assert on `pixels` (2D array) field on `Image` object)?

Reply:
I see you're using `assertNotSame()` below - that should be enough to verify that this is a copy and not the original object right? Or is this to verify that all the properties of the image are still the same?

Reply:
Yeah I wanted to assert on some aspect of the image being the same 🤷